test/index.js

```diff
@@ -79,10 +79,41 @@ describe('broccoli-module-unification-reexporter', function() {
     }
   );
 
+  testExpectation(
+    'should handle initializers', {
+      init: {
+        initializers: {
+          'main.js': 'export default { initialize() { console.log("main one") } };',
+          'i18n.js': 'export default { initialize() { console.log("translations") } };'
+        }
+      }
+    }, {
+      initializers: {
+        'this-addon-name.js': `export { default } from 'this-addon-name/src/init/initializers/main';`,
+        'i18n.js': `export { default } from 'this-addon-name/src/init/initializers/i18n';`,
+      },
+    }
+  );
+
+  testExpectation(
+    'should handle instance-initializers', {
+      init: {
+        'instance-initializers': {
+          'main.js': 'export default { initialize() { console.log("main instance initializer") } };',
+          'i18n.js': 'export default { initialize() { console.log("translation instance initializer") } };'
+        }
+      }
+    }, {
+      'instance-initializers': {
+        'this-addon-name.js': `export { default } from 'this-addon-name/src/init/instance-initializers/main';`,
+        'i18n.js': `export { default } from 'this-addon-name/src/init/instance-initializers/i18n';`,
+      },
+    }
+  );
 
+  // from the RFC it seems like helpers won't be any different from components in this context
+  // https://github.com/dgeb/rfcs/blob/module-unification/text/0000-module-unification.md#generators-and-blueprints
   it('should handle helper reexports');
-  it('should handle initializers');
-  it('should handle instance-initializers');
   it('should handle partials?');
   it('should allow custom mappings as overrides');
```

Comment:
@rwjblue I'm assuming this is in relation to our discussions about "allowing the addon authors to define how their internals are to be remapped in the public api"? This is probably a subject that's best to pair over 😂

Reply:
👍 sounds good to me...